SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -129,7 +163,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bEnableMultiWorker(true)
 	, DefaultRPCRingBufferSize(32)
 	, CrossServerRPCImplementation(ECrossServerRPCImplementation::SpatialCommand)
-	// TODO - UNR 2514 - These defaults are not necessarily optimal - readdress when we have better data
+	// TODO - UNR 2514 - These defaults are not necessarily optimal - readdress bEnableEventTracingRotatingLogswhen we have better data
```

Comment:
Why this change? We (supposedly) want to evaluate ms intervals and all those as well

Reply:
my bad think this was copy+paste